src/js/themes/base.js

```diff
@@ -1082,14 +1083,21 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
     },
     notification: {
+      direction: 'column',
       container: {
         // any box props
         pad: { horizontal: 'small', vertical: 'xsmall' },
         background: {
           color: 'background-front',
         },
       },
+      global: {
```

Comment:
wondering if we should default `global` to `direction: 'row'`?

Reply:
Makes sense, will add.